### DISCUSSION TITLE
Waiting for benefits

### DISCUSSION THREAD

**Discussion by Hector-Du** [community/community#57514] [GitHub Education] [answered]
### Hi everyone,

Hi, I submitted for, and my application was approved, but I still need to receive the benefits.
How long does it last?
Many Thanks
Hector

**Comment by DSDmark** [comment, marked as answer]
Hi @Hector-Du,

According to this discussion:- [**GitHub discussion**](https://github.com/orgs/community/discussions/55559#discussioncomment-5917825)

> when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within **72 hours** of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

And also remember:

- The GitHub Student Developer Pack is free for 2 years. Once your 2 years are up, you can reapply for the pack if you're still a student.
- And I suggest you to read this, there are all things you need to know: [**GitHub Education Repo**](https://github.com/education/Student-Developer-Pack-Current-Partners-FAQ)
